app/Http/Requests/UpdateCommunicationAndConsultationPreferencesRequest.php

```diff
@@ -54,11 +54,7 @@ public function withValidator(Validator $validator)
         $this->conditionallyRequireContactMethods($validator);
 
         $validator->sometimes('meeting_types', 'required', function ($input) {
-            $interviews = ConsultingMethod::where('name->en', 'Interviews')->first()->id;
-            $focusGroups = ConsultingMethod::where('name->en', 'Focus groups')->first()->id;
-            $workshops = ConsultingMethod::where('name->en', 'Workshops')->first()->id;
-
-            return in_array($interviews, $input->consulting_methods ?? []) || in_array($focusGroups, $input->consulting_methods ?? []) || in_array($workshops, $input->consulting_methods ?? []);
+            return in_array('interviews', $input->consulting_methods ?? []) || in_array('focus-group', $input->consulting_methods ?? []) || in_array('workshop', $input->consulting_methods ?? []);
```

Comment:
Another potential way of writing this could be something like.

```php
return $input->consulting_methods && array_intersect(['interviews', 'focus-group', 'workshop'], $input->consulting_methods);
```

However, use whatever approach seems better to you. 

see:
- [`array_intersect`](https://www.php.net/array_intersect)
- https://stackoverflow.com/a/523798 (see comments too as the solution has an error)